py3status/modules/imap.py

```diff
@@ -64,11 +67,30 @@ def check_mail(self):
 
         return response
 
+    def _connection_ssl(self):
+        connection = imaplib.IMAP4_SSL(self.imap_server,
+                                       self.port,
+                                       ssl_context=create_default_context()
```

Comment:
the original module didn't include any `ssl_context` so I think this is safe to remove it.